src/debugger/cordovaDebugSession.ts

```diff
@@ -1275,29 +1275,52 @@ To get the list of addresses run "ionic cordova run PLATFORM --livereload" (wher
 
     private async resolveAndroidTarget(launchArgs: ICordovaLaunchRequestArgs): Promise<string[]> {
         let workingDirectory = launchArgs.cwd;
-        const targetArgs: string[] = [];
+        let targetArgs: string[] = [];
 
         const adbHelper = new AdbHelper(workingDirectory);
         const androidEmulatorManager = new AndroidEmulatorManager(adbHelper);
         const launchScenariousManager = new LaunchScenariosManager(workingDirectory);
 
         const isDevice = launchArgs.target.toLowerCase() === TargetType.Device;
         const isEmulator = launchArgs.target.toLowerCase() === TargetType.Emulator;
+
+        const useDefaultCLI = async () => {
+            this.outputLogger("Continue using standard CLI workflow.");
+            targetArgs = ["--verbose"];
+            const debuggableDevices = await adbHelper.getOnlineDevices();
+            if (debuggableDevices.length) {
+                launchArgs.target = debuggableDevices[0].id;
+            } else {
+                launchArgs.target = TargetType.Emulator;
+            }
```

Comment:
May we use the ternary operator here?
```suggestion
            launchArgs.target = debuggableDevices.length ? debuggableDevices[0].id : TargetType.Emulator; 
```